docs/src/main/paradox/scala/http/server-side/websocket-support.md

```diff
@@ -116,6 +115,20 @@ In case you need to keep inactive connections alive, you can either tweak your i
 'keep-alive' messages regularly.
 @@@
 
+### Handling websockets with independent Source and Sink
+
+Sometimes it is interesting to be able to expose an independent Source and Sink to such WebSocket handler rather than a linear Flow. This is doable by using the `Flow.fromSinkAndSource` or `Flow.fromSinkAndSourceCoupled` methods in Akka 2.5.x (or `CoupledTerminationFlow` in Akka 2.4.x), and passing such composite Flow into the @java[`handleWebSocketRequestWith`]@scala[`handleWebSocketRequestWith`] method. Using these methods you can for example use a `Sink.foreach` to side-efect or print incoming user messages, while providing some kind of data source via a `Source` to the outgoing side of the connection. 
```

Comment:
'This is doable' - perhaps 'This can be done'?
Perhaps don't mention how it was done in 2.4.x?
Might be nice to use `@javadoc`/`@scaladoc` links here?

Reply:
good one.

The reason with the 2.4/2.5 thingy is because if people don't depend explicitly on 2.5, they won't find that type, because akka http still pulls in 2.4... :\ I don't have a better idea on how to write this without confusing people (one or the other group)

Reply:
aah yes until we have akka-http depend on akka 2.5 by default this makes sense indeed